packages/components/forms/stories/ReactHookFormControlledInputs.stories.tsx

```diff
@@ -0,0 +1,267 @@
+        )}
+      </FormControl>
+
+      <Button variant="primary" type="submit">
+        Submit
+      </Button>
+    </Form>
+  );
+};
+
+export const WithSelect = () => {
+  const { control, handleSubmit } = useForm({ mode: 'onChange' });
+
+  const onSubmit = (data) => action('submitted data')(data);
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <FormControl>
+        <FormControl.Label>City</FormControl.Label>
+
+        <ControlledSelect control={control} name="city" defaultValue="">
+          <Select.Option value="" isDisabled>
+            Please, select your city
+          </Select.Option>
+          <Select.Option value="Berlin">Berlin</Select.Option>
+          <Select.Option value="San Francisco">San Francisco</Select.Option>
+          <Select.Option value="Denver">Denver</Select.Option>
+        </ControlledSelect>
+      </FormControl>
+
+      <Button variant="primary" type="submit">
+        Submit
+      </Button>
+    </Form>
+  );
+};
+
+export const WithRadioGroup = () => {
+  const { control, handleSubmit } = useForm({ mode: 'onChange' });
+
+  const onSubmit = (data) => action('submitted data')(data);
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <FormControl>
+        <FormControl.Label>Favorite fruit</FormControl.Label>
+
+        {/**
+         * We need to pass an empty string as default value otherwise react will trigger the uncontrolled inputs warning
+         * more infomation can be found in react-hook-form docs https://react-hook-form.com/api/usecontroller
+         */}
+        <ControlledRadioGroup
+          control={control}
+          name="favorite-fruit"
+          defaultValue=""
+        >
+          <Radio id="apples" value="apples">
+            Apples
+          </Radio>
+          <Radio id="pears" value="pears">
+            Pears
+          </Radio>
+          <Radio id="peaches" value="peaches">
+            Peaches
+          </Radio>
+        </ControlledRadioGroup>
+      </FormControl>
+
+      <Button variant="primary" type="submit">
+        Submit
+      </Button>
+    </Form>
+  );
+};
+
+export const WithCheckboxGroup = () => {
+  const { handleSubmit, control } = useForm();
+  const onSubmit = (data) => action('submitted data')(data);
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)}>
+      <FormControl>
+        <FormControl.Label>Favorite fruits</FormControl.Label>
+
+        <CheckboxGroup>
+          <ControlledCheckbox
+            control={control}
+            id="apples"
+            name="apples"
+            value="apples"
+          >
+            Apples
+          </ControlledCheckbox>
+
+          <ControlledCheckbox
+            control={control}
+            id="pears"
+            name="pears"
+            value="pears"
+          >
+            Pears
+          </ControlledCheckbox>
+
+          <ControlledCheckbox
+            control={control}
+            id="peaches"
+            name="peaches"
```

Comment:
Shouldn't all the checkboxes have the same name??

Reply:
Yes! My bad!